Example/Example/AppDelegate.swift

```diff
@@ -20,14 +29,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Initialize the RoverCampaigns SDK with all modules.
-        RoverCampaigns.initialize(assemblers: [
+        RoverFoundation.initialize(assemblers: [
```

Comment:
Breaking change then?

Reply:
@seanrucker right so let me explain.

Believe it or not this is not a breaking change.

Firstly, users handle imports of our SDK modules with SwiftPM the same way as they did with Carthage: import each module independently.

For Cocoapods users, there is no change.

The example app itself has changed, because the Example app has gone from using Cocoapods to using the SwiftPM method. If the Example app had been (somehow) using Carthage before, then these code changes would have been unnecessary.

Reply:
For even further clarification, here is what happens depending on what app developer does:

Carthage switch to SwiftPM -> no code changes
Cocoapods staying on Cocoapods -> no code changes
Cocoapods to SwiftPM -> code changes to add the imports.

Reply:
It may be worth us, after FanReach is happily settled on SwiftPM, doing a quick writeup for Yinz (and others) to assist them with going from Cocoapods to SwiftPM.

Reply:
Sounds good.